<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManageDialog</class>
 <widget class="QDialog" name="UserManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User management</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resuurces/resources.qrc">
    <normaloff>:/edit-find-user.png</normaloff>:/edit-find-user.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="userList"/>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addUserBtn">
        <property name="toolTip">
         <string>Add user to database</string>
        </property>
        <property name="whatsThis">
         <string>Add user button</string>
        </property>
        <property name="text">
         <string>Add user</string>
        </property>
        <property name="icon">
         <iconset resource="resuurces/resources.qrc">
          <normaloff>:/list-add-user.png</normaloff>:/list-add-user.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delUserBtn">
        <property name="toolTip">
         <string>Remove user from database</string>
        </property>
        <property name="whatsThis">
         <string>Delete user button</string>
        </property>
        <property name="text">
         <string>Delete user</string>
        </property>
        <property name="icon">
         <iconset resource="resuurces/resources.qrc">
          <normaloff>:/list-remove-user.png</normaloff>:/list-remove-user.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pswrdBtn">
        <property name="toolTip">
         <string>Set user password</string>
        </property>
        <property name="whatsThis">
         <string>Set user password button</string>
        </property>
        <property name="text">
         <string>User password</string>
        </property>
        <property name="icon">
         <iconset resource="resuurces/resources.qrc">
          <normaloff>:/document-encrypt.png</normaloff>:/document-encrypt.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+P</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>185</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="User add">
   <property name="icon">
    <iconset resource="resuurces/resources.qrc">
     <normaloff>:/list-add-user.png</normaloff>:/list-add-user.png</iconset>
   </property>
   <property name="text">
    <string>actionUserAdd</string>
   </property>
   <property name="toolTip">
    <string>User add</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resuurces/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserManageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addUserBtn</sender>
   <signal>clicked()</signal>
   <receiver>UserManageDialog</receiver>
   <slot>onUserAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delUserBtn</sender>
   <signal>clicked()</signal>
   <receiver>UserManageDialog</receiver>
   <slot>onUserDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pswrdBtn</sender>
   <signal>clicked()</signal>
   <receiver>UserManageDialog</receiver>
   <slot>onUserPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>UserManageDialog</receiver>
   <slot>onSelectedUserChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUserAdd()</slot>
  <slot>onUserDel()</slot>
  <slot>onUserPassword()</slot>
  <slot>onSelectedUserChanged(int)</slot>
 </slots>
</ui>
